app/liquid_tags/podcast_tag.rb

```diff
@@ -33,7 +33,7 @@ def initialize(_tag_name, link, _parse_context)
   end
 
   def render(_context)
-    ActionController::Base.new.render_to_string(
+    ApplicationController.render(
```

Comment:
Had to make this change to this portion can have access to helpers. I'm going to make this change to the rest of liquid tag in a different PR.

Reply:
**suggestion (non-blocking):** I think `ApplicationController.render` reads nicer than `ActionController::Base.new.render_to_string`, but if you don't want to change all Liquid tags, you should be able to `include ApplicationHelper` in the base class (`LiquidTagBase`) and be done with it. Just to give you an option for the refactor PR.